api/v1/app.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+"""make api"""
+from models import storage
+from os import getenv
+from api.v1.views import app_views
+from flask import Flask, make_response, jsonify
+
+
+app = Flask(__name__)
```

Comment:
Turns out it's a Flask config value. In `api/v1/app.py`, we can add `app.config['JSONIFY_PRETTYPRINT_REGULAR'] = True` in the line after `app`'s assignment to get the same printing of dictionaries as the task examples.